### PR TITLE
Add list option to blacklist and whitelist

### LINKED ISSUE #442: Idea - list whitelist / blackist from the command line
Now that we have a pihole command that will perform things based on input I was checking out the whitelist/blacklist items (to update the FAQ later).

I noticed there is an option to add or remove items from the list, but not to list the items in the list. Could `sudo pihole -w -l` or `sudo pihole -w --list` be added where it will simply display the contents of the black- or whitelist ?

### PR DESCRIPTION
**Pull request by 7h3ju57**
Fixes #442 

Changes proposed in this pull request:

- Added an option to display blacklist and whitelist with option "-l" or "--list"

@pi-hole/gravity

### CODE REVIEW

File: advanced/Scripts/blacklist.sh [hunk 1]
```diff
@@ -61,6 +61,7 @@ function helpFunc()
     echo ":::  -f, --force			Force updating of the hosts files, even if there are no changes"
     echo ":::  -q, --quiet			output is less verbose"
     echo ":::  -h, --help			Show this help dialog"
+    echo ":::  -l, --list           Display your blacklisted domains"
```
Comment (Mcat12) [inline advanced/Scripts/blacklist.sh]: Use tabs here to keep in line with the other options

File: advanced/Scripts/blacklist.sh [hunk 2]
```diff
@@ -173,6 +174,19 @@ function Reload() {
...
 }
 
+function DisplayBlist() {
+clear
+echo -e " Displaying Gravity Affected Domains \n"
+
+GRD="$blacklist"
+count=1
+while IFS= read -r RD
+do
```
Comment (Mcat12) [inline advanced/Scripts/blacklist.sh]: Please indent either 4 spaces or 1 tab

File: advanced/Scripts/whitelist.sh [hunk 3]
```diff
@@ -60,6 +60,7 @@ function helpFunc()
     echo ":::  -f, --force			Force updating of the hosts files, even if there are no changes"
     echo ":::  -q, --quiet			output is less verbose"
     echo ":::  -h, --help			Show this help dialog"
+    echo ":::  -l, --list           Display your whitelisted domains"
```
Comment (Mcat12) [inline advanced/Scripts/whitelist.sh]: Same here, tabs

File: advanced/Scripts/whitelist.sh [hunk 4]
```diff
@@ -186,6 +187,19 @@ function Reload() {
...
+echo -e " Displaying Gravity Resistant Domains \n"
+
+GRD="$whitelist"
+count=1
+while IFS= read -r RD
+do
+  echo "${count}: $RD"
+count=$((count+1))
+done < "$whitelist"
+}
```
Comment (Mcat12) [inline advanced/Scripts/whitelist.sh]: Indent please. It really helps a lot when you read code. ;)

**Comment by PromoFaux** [comment]
To echo @Mcat12, we'll need you tidy up the indenting/spacing to match the rest of the file before we can accept the PR. 

Other than that, great idea! I meant to implement it myself the other day, and then life got in the way....

**Comment by 7h3ju57** [comment]
Sure thing must have missed it

**Comment by PromoFaux** [comment]
Looks good, there are a couple of tweaks I'm going to make to it, but it works! Thanks again!

**Pull request merged**